internal/provider/node_room_member.go

```diff
@@ -43,9 +59,11 @@ func (s *nodeRoomMemberResource) Metadata(ctx context.Context, req resource.Meta
 func (s *nodeRoomMemberResource) Schema(ctx context.Context, req resource.SchemaRequest, resp *resource.SchemaResponse) {
 	resp.Schema = schema.Schema{
 		Description: `
-Provides a Netdata Cloud Node Room Member resource. Use this resource to manage node membership to the room in the selected space, only reachable nodes can be added to the room.
-This resource is useful in the case of [Netdata Streaming and Replication](https://learn.netdata.cloud/docs/observability-centralization-points/metrics-centralization-points/) when you want to spread
-the Netdata child agents across different rooms because by default all of them end in the same room like the Netdata parent.`,
+Provides a Netdata Cloud Node Room Member resource. Use this resource to manage node membership to the room in the selected space.
+There are two option to add nodes to the room:
```

Comment:
```suggestion
There are two options to add nodes to the room:
```

Reply:
don't use dots at the end of a point. https://service-manual.ons.gov.uk/content/formatting-and-punctuation/lists (I'm really not tryting to be an asshole :D )